pkgs/os-specific/darwin/apple-sdk-11.0/default.nix

```diff
@@ -61,7 +61,9 @@ let
   mkStdenv = stdenv:
     if stdenv.isAarch64 then stdenv
     else
+
```

Comment:
```suggestion
```

Reply:
I pushed an updated commit without the newline. I’m not sure how it ended up there, but it’s gone.